csharp/Program.cs

```diff
@@ -1,12 +1,59 @@
 ﻿using System;
 
-namespace csharp
+namespace Cardnumber
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello C#!");
+            Console.WriteLine("Hello World!");
+            Console.WriteLine(GetIssuer(4111111111111111));// == "VISA"
+            Console.WriteLine(GetIssuer(4111111111111111));// == "VISA"
+            Console.WriteLine(GetIssuer(4111111111111));// == "VISA"
+            Console.WriteLine(GetIssuer(4012888888881881));// == "VISA"
+            Console.WriteLine(GetIssuer(378282246310005));// == "AMEX"
+            Console.WriteLine(GetIssuer(6011111111111117));// == "Discover"
+            Console.WriteLine(GetIssuer(5105105105105100));// == "Mastercard"
+            Console.WriteLine(GetIssuer(5105105105105106));// == "Mastercard"
+            Console.WriteLine(GetIssuer(9111111111111111));// == "Unknown"*/
+            Console.ReadKey();
+        }
+        static string GetIssuer(Int64 card)
+        {
+            string num;
+            int num_aux;
+            num = Convert.ToString(card);
```

Comment:
If the input is converted to `string`, why directly defining `card` as `string` directly in function parameter definition? Also, In cases like this, maybe it's better if you assign the value in the same line where the variable is defined, do you have any reason to keep those separated?